### DISCUSSION TITLE
FaceDetailer

### DISCUSSION THREAD

**Discussion by VeilanceDev** [ambocclusion/ComfyUI-SDXL-DiscordBot#6] [Ideas]
Hi! First I just want to say thank you for your work with this fork! I was wondering if you have any plans to add FaceDetailer support? I find that it can make a big difference in image quality.

**Comment by ambocclusion** [comment]
Hey! I've thought about this but I've not been able to come up with a good UX for it. Now that the new edit modals are in perhaps this is where it could live. Let me give it some thought!
